does not build on alpine:latest - update to s3fs version 1.93

```
#6 20.18 g++ -DHAVE_CONFIG_H -I. -I..  -I/usr/include/fuse -D_FILE_OFFSET_BITS=64 -I/usr/include/libxml2    -g -O2 -Wall -fno-exceptions -D_FILE_OFFSET_BITS=64 -D_FORTIFY_SOURCE=2 -MT s3fs_global.o -MD -MP -MF .deps/s3fs_global.Tpo -c -o s3fs_global.o s3fs_global.cpp
#6 20.59 mv -f .deps/s3fs_global.Tpo .deps/s3fs_global.Po
#6 20.59 g++ -DHAVE_CONFIG_H -I. -I..  -I/usr/include/fuse -D_FILE_OFFSET_BITS=64 -I/usr/include/libxml2    -g -O2 -Wall -fno-exceptions -D_FILE_OFFSET_BITS=64 -D_FORTIFY_SOURCE=2 -MT s3fs_help.o -MD -MP -MF .deps/s3fs_help.Tpo -c -o s3fs_help.o s3fs_help.cpp
#6 21.01 In file included from s3fs_help.cpp:26:
#6 21.01 common.h:34:14: error: 'int64_t' does not name a type
```

I didn't dig deep into the issue, because the easy fix is to update s3fs to version 1.93

--- a/Dockerfile
+++ b/Dockerfile
@@ -11,7 +11,7 @@ ENV S3_REGION ''
 VOLUME /var/s3
 
 
-ARG S3FS_VERSION=v1.91
+ARG S3FS_VERSION=v1.93
 
 RUN apk --update add bash fuse libcurl libxml2 libstdc++ libgcc alpine-sdk automake autoconf libxml2-dev fuse-dev curl-dev git; \
     git clone https://github.com/s3fs-fuse/s3fs-fuse.git; \

